### DISCUSSION TITLE
Support older macOSes

### DISCUSSION THREAD

**Discussion by DanielSmedegaardBuus** [MonitorControl/MonitorControl#1359] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

The current version compatible with my macOS 10.13.6 system (2.1.0) doesn't support my LG C2 OLED display.

### Describe the solution you'd like

Just assuming that one of the newer versions does support my TV, it would be really nice if the releases were built to support older macOSes as well :)

### Describe alternatives you've considered

Not sure what is meant by alternatives...

**Comment by waydabber** [comment]
The problem is the LG C2 does not support DDC, so no app update can fix that. If you can at least upgrade to Mojave (10.14), BetterDisplay has support for that and can give you colortable based brightness control which should work with OLED TVs.

**Reply by DanielSmedegaardBuus** [reply]
Oh, okay, that's a shame.

On Windows I use https://github.com/Maassoft/ColorControl - but he's using some network API to control everything on LG TVs.

I guess that's probably out of scope for MonitorControl?

If it is, though, I've already considered making it a project of mine to try and see if I can figure out what kind of communication that app (ColorControl) is doing, and see if I can replicate it myself on macOS. If I could figure that out, it might be something I could contribute with here - but only if I can build on macOS 10.13 (can't upgrade to 10.14 - Mac mini Server 2011).

That leaves me still wondering if the reason for newer versions of your software requires newer versions of OSes simply because you yourselves are upgrading to newer Xcodes, or if there's a technical limitation that means it couldn't ever be built for something like High Sierra?

Thanks for your feedback :)
